testing/run_tests.py

```diff
@@ -232,7 +232,6 @@ def RunDartTest(build_dir, dart_file, verbose_dart_snapshot, multithreaded):
   SnapshotTest(build_dir, dart_file, kernel_file_output, verbose_dart_snapshot)
 
   command_args = [
-    '--disable-observatory',
```

Comment:
I remember we had to add this because we ran into issues because of port collisions causing flakes on some platforms. Instead of enable observatories for all runs, can we just enable it for the one test you added?

Reply:
Ahh ok. It worked locally, so I was hoping it was just ok. I'll split these up.